packages/wallets/src/components/WalletTransfer/WalletTransfer.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import WalletTransferForm from '../WalletTransferForm/WalletTransferForm';
+import './WalletTransfer.scss';
+
+const WalletTransfer = () => {
+    return (
+        <div className='wallets-transfer'>
+            <WalletTransferForm />
+        </div>
+    );
+};
```

Comment:
```suggestion
const WalletTransfer = () => {
    return (
        <div className='wallets-transfer'>
            <WalletTransferForm />
        </div>
    );
};
```
Needs explanation - why is the component only wrapping another component with a div?

Reply:
i was creating the parent component at first and started to create the children based on figma, i didnt realize it just become a wrapper. i thought there will be another component besides form but i think i can just move the form component here